Files/ViewModels/SettingsViewModel.cs

```diff
@@ -92,8 +92,15 @@ public GridLength SidebarWidth
         public async void DetectQuickLook()
         {
             // Detect QuickLook
-            ApplicationData.Current.LocalSettings.Values["Arguments"] = "StartupTasks";
-            await FullTrustProcessLauncher.LaunchFullTrustProcessForCurrentAppAsync();
+            try
+            {
+                ApplicationData.Current.LocalSettings.Values["Arguments"] = "StartupTasks";
```

Comment:
Perhaps you can add here `localSettings.Values["quicklook_enabled"] = 0` so if something goes wrong we consider quicklook not enabled